src/ralph/api/__init__.py

```diff
@@ -8,13 +10,29 @@
 from .auth import AuthenticatedUser, authenticated_user
 from .routers import health, statements
 
+
+def filter_transactions(event, hint):  # pylint: disable=unused-argument
+    """Filter transactions for Sentry."""
+    url_string = event["request"]["url"]
+    parsed_url = urlparse(url_string)
+
+    if settings.SENTRY_IGNORE_HEALTH_ROUTES and parsed_url.path in [
+        "/__lbheartbeat__",
+        "/__heartbeat__",
+    ]:
+        return None
```

Comment:
```suggestion
    from ralph.api.routers.health import router
    # This can be LRU-cached somewhere
    health_check_routes = [route.path for route in router.routes]
    if settings.SENTRY_IGNORE_HEALTH_CHECKS and url.path in health_check_routes:
        return None
```

Reply:
Nice!